android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -2184,18 +2184,14 @@ public boolean getPreventsDisplaySleepDuringVideoPlayback() {
         return preventsDisplaySleepDuringVideoPlayback;
     }
 
-    private void updateFullScreenButtonVisbility() {
+    private void updateFullScreenButtonVisibility() {
         if (playerControlView != null) {
             final ImageButton fullScreenButton = playerControlView.findViewById(R.id.exo_fullscreen);
-            if (controls) {
-                //Handling the fullScreenButton click event
-                if (isFullscreen && fullScreenPlayerView != null && !fullScreenPlayerView.isShowing()) {
```

Comment:
why did you remove this section ?
Please check git history, but I think it was fixing a problem !